e2e/nomostest/clusters/gke.go

```diff
@@ -193,6 +193,7 @@ func createGKECluster(t testing.NTB, name string) error {
 	args = append(args, name,
 		"--project", *e2e.GCPProject,
 		"--async",
+		"--cluster-ipv4-cidr", "/19", // use smaller than default CIDR
```

Comment:
Note this is the same value as is set by the [current terraform](https://github.com/GoogleContainerTools/kpt-config-sync/blob/main/e2e/testinfra/terraform/modules/e2ecluster/e2ecluster.tf#L25)